srtp/srtp.c

```diff
@@ -1323,6 +1295,46 @@ srtp_err_status_t srtp_stream_init_keys(srtp_stream_ctx_t *srtp,
     return srtp_err_status_ok;
 }
 
+srtp_err_status_t srtp_stream_init_all_master_keys(srtp_stream_ctx_t *srtp,
+                                                   const srtp_policy_t *p)
+{
+    srtp_err_status_t status = srtp_err_status_ok;
+    if (p->key != NULL) {
+        if (p->use_mki) {
```

Comment:
Would a single key be a reason to not use MKI? Perhaps pointless, but illegal?

Reply:
I think we will end up removing the single key option in the policy, it was only left there for ABI compatibility when MKI was first added.
It is "illegal" today as the single key option does not contain a mki id.